product.json

```diff
@@ -23,65 +23,5 @@
 	"privacyStatementUrl": "https://privacy.microsoft.com/en-us/privacystatement",
 	"telemetryOptOutUrl": "https://github.com/Microsoft/azuredatastudio/wiki/How-to-Disable-Telemetry-Reporting",
 	"urlProtocol": "azuredatastudio",
-	"enableTelemetry": true,
-	"aiConfig": {
-		"asimovKey": "AIF-444c3af9-8e69-4462-ab49-4191e6ad1916"
-	},
-	"sendASmile": {
-		"reportIssueUrl": "https://github.com/Microsoft/azuredatastudio/issues/new?labels=customer%20reported%20issue",
```

Comment:
Are these being injected somewhere else then? If not why are you taking them out?

Reply:
This entire file is replaced.

Reply:
Replaced during build time or something? Where does this config live now then?

Reply:
Same repo as the icons, replaced as part of the same mixin